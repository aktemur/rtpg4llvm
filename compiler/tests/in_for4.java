/*stmt*/
for(int i=0, k=0; i < 10 && k < 10; i++, k++)
    0;

