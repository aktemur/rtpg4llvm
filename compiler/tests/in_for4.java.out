mkScopedStmt(5, new VarDecl(new IntType(), "i"),
      new VarDecl(new IntType(), "k"),
      new ExprStmt(mkBinOp(Equal,
         new VarExpr("i"),
         mkConst(0))),
      new ExprStmt(mkBinOp(Equal,
         new VarExpr("k"),
         mkConst(0))),
      new WhileStmt(mkBinOp(LogicalAnd,
         mkBinOp(Less,
            new VarExpr("i"),
            mkConst(10)),
         mkBinOp(Less,
            new VarExpr("k"),
            mkConst(10))),
      mkScopedStmt(3, new ExprStmt(mkConst(0)),
      new ExprStmt(mkUnaryPostOp(PlusPlus, new VarExpr("i"))),
      new ExprStmt(mkUnaryPostOp(PlusPlus, new VarExpr("k"))))))
